### ISSUE TITLE
TransformError ENOENT .babelrc in react native 0.17-rc1

### ISSUE THREAD

**Issue by bendyorke** [rackt/react-redux#218]
I ran across a couple issues in the latest version of RN when trying to use react-redux.  These were the errors:

```
react-redux@3.1.2:
TransformError: /.../node_modules/react-redux/lib/components/createAll.js: ENOENT: no such file or directory, open '/.../node_modules/react-redux/.babelrc

react-redux@3.1.0:
TransformError: /.../node_modules/react-redux/lib/components/createAll.js: [BABEL] /.../node_modules/react-redux/lib/components/createAll.js: Unknown option: /.../node_modules/react-redux/.babelrc.stage
```

For now I was able to get around it by cloning the src dir.  I did have to change one ES7 bind operator in `src/components/connect.js:163`.  I'm not sure if it's worth looking into for now, since I hear a lot of awesome changes are coming down the line for RN and it's React dependency ;)

Figured it might help in case anyone else was having the issue!

Also, I was thinking about submitting a pull request, but ran into some issues.  Namely, the `invariant` package doesn't seem to be a dependency, and one of the tests was failing on a fresh copy of `3.x`:
```
1) React connect should throw an error if the store is not in the props or context:
     Error: Expected [Function] to throw 'Invariant Violation: Could not find "store" in either the context or props of "Connect(Container)". Either wrap the root component in a <Provider>, or explicitly pass "store" as a prop to "Connect(Container)".'
      at Object.assert [as default] (node_modules/expect/lib/assert.js:20:9)
      at Expectation.toThrow (node_modules/expect/lib/Expectation.js:90:24)
      at Context.<anonymous> (test/components/connect.spec.js:1080:9)
```

Were those expected?

**Comment by gaearon** [comment]
I'm afraid you need to bring this up with React Native.
We shipped 3.1.2 which doesn't have `.babelrc` (pretty much what your error message says).
If RN assumes every NPM package has a `.babelrc`, it's RN's issue.

I'm happy to reopen and fix if it's us doing something wrong, but right now I don't see anything actionable here.

**Comment by merriam** [comment]
For those arriving here from Google; this error is also caused by caches refusing to clear.  Rebooting the development machine, or killing items in /tmp, will fix this.
